scripts/charts/mongodb.sh

```diff
@@ -13,8 +13,6 @@ NC='\033[0m';
 printf "${YELLOW}Checking for MongoDB-Replicaset with releasename: ${BLUE}mymongodb\n${NC}";
 helm status mymongodb >/dev/null 2>&1 || {
     printf >&2 "${BLUE}Chart not found. Installing MongoDB-Replicaset...\n${NC}";
-    helm repo add stable https://kubernetes-charts.storage.googleapis.com/;
```

Comment:
You should not remove the `helm repo add` command.

Reply:
The `https://kubernetes-charts.storage.googleapis.com/` repo no longer exist (access denied, see [here](https://stackoverflow.com/questions/61954440/how-to-resolve-https-kubernetes-charts-storage-googleapis-com-is-not-a-valid)), `mymongodb` is now available directly in `https://charts.helm.sh/stable` repo.

Reply:
Oh. Thanks for the explanation @llaske